src/app/destinyTrackerApi/bulkFetcher.ts

```diff
@@ -1,11 +1,18 @@
+import { $q, $http } from 'ngimport';
 import { ItemListBuilder } from './itemListBuilder';
+import { TrackerErrorHandler } from './trackerErrorHandler';
+import { ReviewDataCache } from './reviewDataCache';
+import { D1ItemFetchResponse } from '../item-review/destiny-tracker.service';
+import { IPromise } from 'angular';
 
 class BulkFetcher {
-  constructor($q, $http, trackerErrorHandler, loadingTracker, reviewDataCache) {
-    this.$q = $q;
-    this.$http = $http;
+  _reviewDataCache: ReviewDataCache;
+  _loadingTracker: any;
+  _trackerErrorHandler: TrackerErrorHandler;
+  _itemListBuilder: any;
```

Comment:
For the most part you can initialize these here, instead of in the constructor. Like so:

```ts
_itemListBuilder = new ItemListBuilder();
_trackerErrorHandler = new TrackerErrorHandler();
```

Besides being more concise, you don't have to write out their types.